agent/engine/engine_mocks.go

```diff
@@ -17,6 +17,7 @@
 package engine
 
 import (
+	context0 "context"
```

Comment:
I'm using the `context` package in my code.

I have no idea why `golang.org/x/net/context` is still imported here...

Reply:
Looks like the `golang.org/x/net/context` is used by the `task_engine.Init()`, I think that's why the reason it's imported. I asked because I saw two context imported, we can't get rid of this until go docker client updated this.